app/charts/shared/use-sync-interactive-filters.spec.tsx

```diff
@@ -74,24 +74,24 @@ const interactiveFiltersConfig: InteractiveFiltersConfig = {
   },
 };
 
-const chartConfig = migrateChartConfig(
-  {
-    ...fixture.data.chartConfig,
-    interactiveFiltersConfig,
-  },
-  {
-    migrationProps: {
-      meta: {},
-      dataSet: "foo",
-    },
-  }
-) as ChartConfig;
-
-const setup = ({
+const setup = async ({
   modifiedChartConfig,
 }: {
   modifiedChartConfig: ChartConfig;
 }) => {
+  const chartConfig = (await migrateChartConfig(
+    {
+      ...fixture.data.chartConfig,
+      interactiveFiltersConfig,
+    },
+    {
+      migrationProps: {
+        meta: {},
+        dataSet: "foo",
+      },
+    }
+  )) as ChartConfig;
```

Comment:
I think the problem with this is that it's only a `ChartConfig` if we migrate to latest version (which we usually do). Maybe it would be possible to narrow it down somehow 🤔